src/schemas/channel.ts

```diff
@@ -0,0 +1,19 @@
+export interface ChannelPayload {
+  avatar: string,
+  coverUrl: string,
+  desc: string,
+  extras: string,
+  feedType: number,
+  nickname: string,
+  thumbUrl: string,
+  url: string,
+  objectId?: string,
+  objectNonceId?: string,
+}
+
+export enum ChannelType {
+  Unknown = 0,
+  Photo = 2,
+  Video = 4,
+  Live = 9,
```

Comment:
The Live type is very interesting.

If we use the `Post` to contain the `Live` type, it seems we need to design the streaming service for Wechaty.

There are also two more scenarios for the streaming: Video call, and the Audio call.

Reply:
The streaming service sounds very interesting, however so far channel message in wechat does not need such feature. It is just a special link jump to another web page, and contains no streaming feature in chat.